tequilapi/endpoints/proposals.go

```diff
@@ -57,8 +57,20 @@ type serviceDefinitionRes struct {
 	LocationOriginate locationRes `json:"location_originate"`
 }
 
+// swagger:model QualityMetrics
+type proposalsQualityMetricsRes struct {
+	Metrics []qualityMetricsRes `json:"metrics"`
+}
+
+type qualityMetricsRes struct {
+	ProviderID  string `json:"provider_id"`
+	ServiceType string `json:"service_type"`
+	metricsRes
+}
+
 type metricsRes struct {
-	ConnectCount quality.ConnectCount `json:"connect_count"`
+	ConnectCount     quality.ConnectCount `json:"connect_count"`
+	MonitoringFailed bool                 `json:"monitoring_failed"`
```

Comment:
BTW, are you introducing this to Tequilapi `client.go` somehow? We are migrating to same DTOs there at `contract` package.

Reply:
It is in the `contract` package now.